{"name": "test_user_can_log_in[3]", "status": "broken", "statusDetails": {"message": "KeyError: '3'", "trace": "E   KeyError: '3'"}, "parameters": [{"name": "id", "value": "'3'"}], "start": 1644365159185, "stop": 1644365159186, "uuid": "f87cac65-7bf9-436e-ba38-4ea238944ccb", "historyId": "5b03d52b57a9e2035d976e526c289ccf", "testCaseId": "3c0c80c1e8f7586509bf22a553a53fa9", "fullName": "test_login_page#test_user_can_log_in", "labels": [{"name": "tag", "value": "all_test"}, {"name": "suite", "value": "test_login_page"}, {"name": "host", "value": "DESKTOP-UK0K253"}, {"name": "thread", "value": "1268-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_page"}]}